src/bin.rs

```diff
@@ -36,11 +37,21 @@ fn main() -> std::result::Result<(), Box<dyn std::error::Error + Send + Sync>> {
     .format_module_path(false)
     .try_init();
 
-    let host = Host::new();
+    let mut builder = HostBuilder::new();
+    #[cfg(feature = "lattice")]
+    let builder = builder.with_gantryclient(get_gantry_client());
 
-    let manifest = HostManifest::from_path(cmd.manifest_path, cmd.expand_env)?;
-    host.apply_manifest(manifest)?;
-    info!("Processed and applied host manifest");
+    let host = builder.build();
+
+    if let Some(ref mp) = cmd.manifest_path {
+        let manifest = HostManifest::from_path(mp, cmd.expand_env)?;
+        host.apply_manifest(manifest)?;
+        info!("Processed and applied host manifest");
+    } else {
+        info!("Starting without manifest");
+        #[cfg(not(feature = "lattice"))]
+        warn!("Started without manifest and without lattice. This host cannot launch actors or providers.");
```

Comment:
starting without a manifest is something that will be commonplace when people are using lattice.
If someone starts without lattice and without a manifest.. then you're probably right  - we should abort because once that type of host is running, it can't accept a capability provider remotely nor can it accept actors.